virtcontainers/container.go

```diff
@@ -1246,3 +1260,88 @@ func (c *Container) detachDevices() error {
 	}
 	return nil
 }
+
+// creates a new cgroup and return the cgroups path
+func (c *Container) newCgroups() error {
+	ann := c.GetAnnotations()
+
+	config, ok := ann[annotations.ConfigJSONKey]
```

Comment:
We use in many places the config json, at some point better to move it from annotations, virtcontainers is not agnostic at all.

Reply:
Agree. We can definitely save the config in `Container` . This way we can save many disk read and json Unmarshaling.